examples/wdio/mocha/mocha.test.js

```diff
@@ -1,11 +1,11 @@
 const assert = require('assert')
 
-describe('webdriver.io page', () => {
+describe(`${testData.url} page`, () => {
```

Comment:
You should create an own dataprovider example

Reply:
I have created a new file to demonstrate data provider.